astropy/utils/iers/tests/test_iers.py

```diff
@@ -68,6 +68,13 @@ def test_open_filename(self):
         with pytest.raises(FILE_NOT_FOUND_ERROR):
             iers.IERS.open('surely this does not exist')
 
+    def test_open_network_url(self):
+        iers.IERS_A.close()
+        iers.IERS_A.open("file://" + IERS_A_EXCERPT)
```

Comment:
Unfortunately, now this bit is causing the tests to fail on windows.  Should use:
`'file:' + `urllib.request.pathname2url`.  For supporting this on Python 2 and 3 you'll have to use `from ....extern.six.moves import urllib`.